CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+<a name="4.4.0"></a>
+# [4.4.0](https://github.com/akveo/nebular/compare/v4.3.1...v4.4.0) (2019-10-07)
+
```

Comment:
### Highlights
Nebular 4.4.0 features synchronization with Eva 1.1 adding transparent colors.

Reply:
Added in 8783cc21089c3f50bb6c48eb16ce60df7f26fc80